polkadot/runtime/common/src/impls.rs

```diff
@@ -64,6 +64,7 @@ pub fn era_payout(
 	total_stakable: Balance,
 	max_annual_inflation: Perquintill,
 	period_fraction: Perquintill,
+	min_fraction_remainder: Percent,
```

Comment:
Nit: It is a little bit weird to have min_remainder especially considering the docs for era_payout. Personally I would suggest to instead use `max_fraction_stakers`.
```
/// Determine the payout for this era.
///
/// Returns the amount to be paid to stakers in this era, as well as whatever else should be
/// paid out ("the rest").
```

Reply:
Yes, I think this is a good idea. It also simplifies the era payout math. I have changed the PR accordingly.